tpot/base.py

```diff
@@ -684,6 +678,61 @@ def export(self, output_file_name):
         with open(output_file_name, 'w') as output_file:
             output_file.write(export_pipeline(self._optimized_pipeline, self.operators, self._pset))
 
+    def _impute_values(self, features):
+        """Impute missing values in a feature set.
+
+        Parameters
+        ----------
+        features: array-like {n_samples, n_features}
+            A feature matrix
+
+        Returns
+        -------
+        array-like {n_samples, n_features}
+        """
+        if self.verbosity > 1:
+            print('Imputing missing values in feature set')
+
+        def impute_feature(feature):
+            """Impute the missing values in just one feature.
+
+            Parameters
+            ----------
+            feature: array-like {n_samples}
+                An individual feature from a feature matrix.
+
+            Returns
+            -------
+            array-like {n_samples}
+            """
+            return Imputer(strategy="median", axis=1).fit_transform(feature)[0]
+
+        return np.apply_along_axis(impute_feature, axis=0, arr=features)
```

Comment:
This part isn't necessary. `Imputer` is already applied to each column separately.